java/core/src/java/org/apache/orc/impl/RecordReaderImpl.java

```diff
@@ -577,19 +645,21 @@ private static boolean shouldEvaluateBloomFilter(PredicateLeaf predicate,
   private static TruthValue evaluatePredicateMinMax(PredicateLeaf predicate, Object predObj,
       Object minValue,
       Object maxValue,
-      boolean hasNull) {
+      boolean hasNull,
```

Comment:
I think you should make the same change here.

Reply:
Should be fixed in the new commit.